### PR TITLE
Format

### PR DESCRIPTION
**Pull request by Raphael-Mabille**
add the detection of the format of the input file if it was not given

### CODE REVIEW

File: src/ConfUtils.hs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+-- MyPandoc
+-- File description:
+-- ConfUtils
+-}
+
+module ConfUtils (getFomat) where
+
+import Conf (Conf(..), DocumentFormat(..))
+
+getFomat :: Conf -> String -> Conf
```
Comment (ThibautHH) [inline src/ConfUtils.hs:12]: Typo, should be `getFormat`, should rename everywhere.

File: MyPandoc.cabal [hunk 2]
```diff
@@ -26,6 +26,9 @@ source-repository head
 library
   exposed-modules:
```
Comment (ThibautHH) [inline MyPandoc.cabal:27]: This should changed in package.yml, the .cabal file is auto-generated.

**Review by ThibautHH — CHANGES_REQUESTED** [review]

File: src/ConfUtils.hs (hunk 1 above)
Comment (Raphael-Mabille) [reply src/ConfUtils.hs]: done

File: MyPandoc.cabal (hunk 2 above)
Comment (Raphael-Mabille) [reply MyPandoc.cabal:27]: done

**Review by ThibautHH — APPROVED** [review]

**Pull request merged**